crypto/x509/v3err.c

```diff
@@ -1,6 +1,6 @@
 /*
  * Generated by util/mkerr.pl DO NOT EDIT
- * Copyright 1995-2019 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 1995-2020 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
If it is the only change, please remove it.

Reply:
This is autogenerated change with make update , if i remove it travis will fail

Reply:
v3err.c file is not fo human editing

Reply:
Just remove it from your commit.